lib/components/CelebGallery/index.tsx

```diff
@@ -1,52 +1,56 @@
+import {useRouter} from 'next/router';
 
 export const CelebGallery: React.FC<{
   celebGalleryItems: TCelebGalleryItem[];
 }> = (p) => {
   const context = useCelebContext();
+  const router = useRouter()
 
+  // dynamic classes
+  let grid_class = ['grid', 'gap-4', router.pathname == '/' ? 'grid-cols-2' : 'grid-cols-3', router.pathname == '/' ? 'sm:grid-cols-3' : 'sm:grid-cols-4', 'lg:grid-cols-4', 'items-center', 'justify-center', 'w-full'].join(' ')
   return (
-    <div className={s.CelebGallery}>
-      {p.celebGalleryItems.map((celebData) => {
-        const picture = celebData.picture || context.placeholderImage;
+    <div className="container mx-auto w-full h-auto items-center justify-center">
+      <div className={grid_class}>
+        {p.celebGalleryItems.map((celebData) => {
+          const picture = celebData.picture || context.placeholderImage;
 
-        return (
-          <a
-            className={s.link}
-            href={`/${celebData.slug}`}
-            key={celebData.slug}
-          >
-            <span className={s.image}>
-              <Image
-                objectFit="cover"
-                objectPosition="top"
-                blurDataURL={picture.metadata.lqip}
-                placeholder="blur"
-                src={sanityImage(picture).width(200).height(250).url()}
-                alt={celebData.name}
-                layout="fixed"
-                width={160}
-                height={200}
-                className={s.image}
-              />
-            </span>
-            <Typography
-              fontWeight={500}
-              variant="h4"
-              component="p"
-              className={s.name}
+          return (
+            <a
+              className="celeb-card overflow-hidden rounded-xl w-full"
+              href={`/${celebData.slug}`}
+              key={celebData.slug}
             >
-              {celebData.name}
-            </Typography>
-          </a>
-        );
-      })}
+              <span className="relative block w-full h-auto">
+                <Image
+                  objectFit="cover"
+                  objectPosition="center"
+                  blurDataURL={picture.metadata.lqip}
+                  placeholder="blur"
+                  src={sanityImage(picture).url()}
+                  alt={celebData.name}
+                  layout="responsive"
+                  width={200}
+                  height={225}
+                  className="rounded-lg w-full h-auto block"
+                />
+
+                <p className="font-primary absolute bottom-3 left-3 font-semibold text-white z-10 text-sm">
+                  {celebData.name}
+                </p>
+
+                {/* overlay */}
+                <div className="pointer-events-none absolute top-0 left-0 h-full w-full bg-gradient-to-b from-transparent via-transparent to-black block" />
+              </span>
+              
+            </a>
+          );
+        })}
+      </div>
     </div>
   );
 };
```

Comment:
I don't think we got this right yet. I propose the following changes...

* Remove `grid-cols-3`. The 3 celebs per row view is not good.
* Use `grid-cols-2`. For screens larger than `md` use `grid-cols-4`
* Set the max-width of images to 200pixels. Pictures larger than 200pixels don't look good.
* Center the images.

What do you think?

Reply:
I set the max-width of the images to 200 pixels, the gap between the celeb galleries on a large screen is different because of the max-width<br>

### lg:
It have some very small white left and right because of the max-width.<br>
![image](https://user-images.githubusercontent.com/68635840/165452970-40ef4577-f942-4fde-9f90-638f1ebe21c4.png)

### md:<br>
![image](https://user-images.githubusercontent.com/68635840/165453070-2ccd24d6-3e28-464b-a101-189844dedb68.png)

### mobile view
grid-cols-2 but I removed the 200 pixel max-width in mobile view because it's taking a lot of spaces and it's not looking good.<br>
![image](https://user-images.githubusercontent.com/68635840/165453505-0c905fc7-a60e-45cf-a686-1368ca84cd4a.png)

Reply:
Okay, but take a look at Interesting Profiles section on the celeb page. Set the browser width to 630. You'll see that the images are comically gigantic on that view, right?